srcpkgs/libopensmtpd/template

```diff
@@ -0,0 +1,31 @@
+# Template file for 'libopensmtpd'
+pkgname=libopensmtpd
+version=0.7
+revision=1
+hostmakedepends="mdocml"
+makedepends="libevent-devel"
+short_desc="Library for creating opensmtpd filters"
+maintainer="Adam Gausmann <agausmann@fastmail.com>"
+license="custom:BSD-like"
```

Comment:
`ISC` is fine for this, http://imperialat.at/dev/libopensmtpd/LICENSE https://spdx.org/licenses/ISC.html